reco_utils/recommender/rlrmc/RLRMCalgorithm.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import random
```

Comment:
apart from the tests in the notebooks, we also have unit tests. If you have time and it is not too much work, it would be nice to have unit tests for the main classes of RLRMC. Here you can see an example: https://github.com/Microsoft/Recommenders/blob/staging/tests/unit/test_ncf_singlenode.py